src/index.ts

```diff
@@ -141,11 +145,25 @@ async function run() {
   }
 }
 
-log("Starting service");
 
-cron.schedule(config.cron, run);
+function init() {
+  console.log('init - why not use pino')
```

Comment:
use `function log(message: string)` to log messages.

Reply:
Can't log before the config is loaded.

Also, I would but I was just following the patterns already present

https://github.com/rmontgomery2018/rssGetto/blob/main/src/index.ts#L58
https://github.com/rmontgomery2018/rssGetto/blob/main/src/index.ts#L132